lib/DXIL/DxilModule.cpp

```diff
@@ -380,10 +359,6 @@ void DxilModule::CollectShaderFlagsForModule(ShaderFlags &Flags) {
   bool hasCSRawAndStructuredViaShader4X =
       hasRawAndStructuredBuffer && m_pSM->GetMajor() == 4 && m_pSM->IsCS();
   Flags.SetCSRawAndStructuredViaShader4X(hasCSRawAndStructuredViaShader4X);
-
-  if (!Flags.GetRaytracingTier1_1()) {
-    Flags.SetRaytracingTier1_1(RequiresRaytracingTier1_1(GetSubobjects()));
-  }
```

Comment:
Without this, the only indication of raytracing tier 1.1 requirement is the opcodes check that I referenced in my global comment.

Reply:
And that's by-design.  The subobject dependency on DXR 1.1 is independently checked by the runtime.  If this flag is not set on the module, then nothing in the module sent to the driver has a DXR 1.1 dependency.  Only if the runtime uses a subobject with that dependency (and checks the requirement), will it send a subobject to the driver requiring DXR 1.1.